routing/result_interpretation.go

```diff
@@ -26,6 +26,20 @@ type pairResult struct {
 	success bool
 }
 
+// newFailPairResult creates a new result struct for a failure.
+func newFailPairResult(minPenalizeAmt lnwire.MilliSatoshi) pairResult {
```

Comment:
nit: maybe `new` is redundant? Could do `failPairResult(99)` and `successPairResult` only

Reply:
renamed